src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -365,15 +365,16 @@ LedgerMaster::getFullValidatedRange (std::uint32_t& minVal, std::uint32_t& maxVa
     if (!maxVal)
         return false;
 
+    boost::optional<std::uint32_t> maybeMin;
```

Comment:
I'm going to punt on that for this PR.  I think its worth considering, but better as a refactor/simplification of `LedgerMaster`.